src/gui/windows/debug_network.zig

```diff
@@ -38,16 +38,22 @@ pub fn render() void {
 		const loss = @as(f64, @floatFromInt(resent))/@as(f64, @floatFromInt(sent))*100;
 		draw.print("Packet loss: {d:.1}% ({}/{})", .{loss, resent, sent}, 0, y, 8, .left);
 		y += 8;
+		draw.print("Internal message overhead: {}kiB", .{network.Connection.internalMessageOverhead.load(.monotonic) >> 10}, 0, y, 8, .left);
```

Comment:
Some thoughts on using kiB - we get into big numbers really quickly, 6 digit values for hundreds of MiB are hard to read. Maybe we could show `{}kiB ({}MiB)` or just use MiB and have few decimal places?

Reply:
Yeah, I agree. But on the other hand there are protocols that only use a few kilobytes in total.
So I guess maybe a dynamic approach would be best,

Reply:
I tried dynamic scales, but having numbers of different scales makes it more confusing in my opinion.
A separation character would be nice, but sadly Zig's formatter doesn't support this.

> Maybe we could show {}kiB ({}MiB)

The text is already too wide in my opinion, so I'd like to avoid adding more characters.